pylxd/client.py

```diff
@@ -130,6 +130,8 @@ def __init__(self, endpoint=None, version='1.0', cert=None, verify=True):
             auth = response.json()['metadata']['auth']
             if auth != "trusted":
                 raise exceptions.ClientAuthenticationFailed()
+
```

Comment:
I'd remove the extra line here. No biggie one way or the other though, if you feel its important.